tests/phpunit/tests/term/getTerms.php

```diff
@@ -2635,6 +2635,19 @@ public function test_pad_counts_should_work_when_first_taxonomy_is_nonhierarchic
 		}
 	}
 
+	/**
```

Comment:
Why this is broken:

* The factory creates a post with a status of published
* `wp_insert_post()` ultimately calls `wp_set_object_terms()` which increments the count as the post is published
* The count is then incremented again on the call to `_update_term_count_on_transition_post_status()`
* Off by one...